index.html

```diff
@@ -4,9 +4,88 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header>
+        <!-- <div id="clock">
+            clock
+        </div>  -->
```

Comment:
Style: It's an industry best practice to remove commented out code before submitting a PR.